v2/src/Interfaces.ts

```diff
@@ -0,0 +1,16 @@
+export interface TransitIconImages {
+  [gotrain: string]: string,
+  via: string,
+  streetcar: string,
+  subway: string,
+  bus: string
+}
+
+export interface ActiveTransit {
+  [gotrain: string]: boolean,
```

Comment:
```suggestion
  gotrain: boolean,
```

Was this the string indexing thing you were talking about?

Reply:
yeah, if you don't flag the key as a string, it doesn't let you index with a string lmao

Reply:
Why do you need to index it with a string?

Reply:
On the function generateIcon I use the icon object with all the URLs